### DISCUSSION TITLE
Opaque box sizing

### DISCUSSION THREAD

**Discussion by strangenick2** [SubtitleEdit/subtitleedit#6609] [Ideas]
I need a bigger Opaque box when embedding subtitles, but I can't find setting for that. I can't cover other embedded subtitles at the moment and it's very annoying. Other times I wish opaque box to be edge to edge on the screen. Is it possible to add a Opaque box size setting?

**Comment by niksedk** [comment]
Could you explain a bit more? What are you doing and where are you doing it? What subtitle format?
https://www.nikse.dk/subtitleedit/help#good-github-issue

**Comment by strangenick2** [comment]
I'm using .ass subtitle format. I'm using Opaque box for generate video with burned-in subs (first photo), because sometimes there are other burned-in subtitles on the video file and I wish to hide them. Currently the Opaque box sticks out only 1-2 mm aside and this cannot be changed (second photo). I would like it to come out more - 5 or 10 mm (third photo - from other program). Simetimes I'm translating shows from an American TV station that constantly plays text and ads at the bottom of the screen, which are very distracting - then I'd like to put a black Opaque box across the entire bottom of the screen for my subtitles to appear on. However, for this I need to be able to set Opaque box options - more precisely its size (left, right, top, bottom of the Opaque box). 
![1](https://user-images.githubusercontent.com/123251046/213871682-4e017d42-ddb9-4323-bbe0-3c5370a70af7.JPG)
![2](https://user-images.githubusercontent.com/123251046/213871699-d206db8e-cb3d-4646-9e9d-4ab5da23d9ca.JPG)
![3](https://user-images.githubusercontent.com/123251046/213871703-df65e494-74d7-46b4-8141-5856e159d6c0.JPG)

**Comment by niksedk** [comment]
It's not possible with the default ASSA opaque box... but you can try this:
![image](https://user-images.githubusercontent.com/1261331/213873390-ed6a5a31-f963-42bb-a9ea-13fef6ad2dc2.png)

See more here: https://www.nikse.dk/subtitleedit/help#assa_bg_box

Note: This needs to be the absolutely last step, as the boxes will not expand/collapse if you change texts.

Works on selected lines (so you can use e.g. ctrl+a in list view to select all lines first).

**Reply by strangenick2** [reply]
Thank you! I will try it!

**Reply by strangenick2** [reply]
Everything is perfect! Much better than I imagined! Thank you very much!
